### ISSUE TITLE
Hide top/bottom toolbars on mobile devices

### ISSUE THREAD

**Issue by jan-kleks** [KyleU/solitaire.gg#3]
Even on devices with large screens such as the iPhone 7 Plus everything feels crammed. Hiding top and bottom toolbars in some way would definitely help.

Keep up good work!
Jan

**Comment by KyleU** [comment]
Agreed! You can double-click the background to show/hide the bars, but that's not very obvious, obviously ;)

Any ideas for a better way to expose this?

**Comment by jan-kleks** [comment]
Almost like an easter-egg. :P Maybe use this "question mark icon" (add a separate icon for rules) for tips regarding solitaire.gg itself.

**Comment by jan-kleks** [comment]
On my iPhone 7 it's sometimes hard to use this "double-tap". It does not seem to work every time.  Especially on the right side of the screen. But maybe this is a Safari issue...

**Comment by KyleU** [comment]
The new ui only has one bar, displayable at the top or bottom. I can hide the bar on mobile, but I still need some way for a mobile user (without a keyboard or gamepad) to access the menus... any idea?

**Comment by jan-kleks** [comment]
Hmm... Double-tapping to hide the bar is not that bad (except for the issues I've described here https://github.com/KyleU/solitaire.gg/issues/3#issuecomment-291308482), and it could work for this new one-bar interface too. You may also experiment with gestures like swiping. 

The three-bar symbol can be used for the menu, and it should look fine on mobile devices as well. Check whether the menu on mobile devices wouldn't look better if displayed fullscreen.
